projects/core/src/forms/control-label/control-label.element.scss

```diff
@@ -25,12 +25,15 @@
   text-transform: var(--text-transform) !important;
   cursor: pointer !important;
 
-  @include line-height-eraser(
-    getLineHeightGap($cds-global-typography-secondary-line-height-static, $cds-global-typography-secondary-line-height),
-    $cds-global-typography-top-gap-height,
-    $cds-global-typography-ascender-height,
-    $cds-global-typography-x-height
-  );
+  .cds-required-mark {
```

Comment:
I might be missing something, but this doesn't seem to work. I don't know much about shadow DOM and slots, but I am seeing if I can figure it out. I think this might need to be in the global CSS, not the shadow CSS.